api/src/main/java/org/openmrs/module/pihcore/config/ConfigDescriptor.java

```diff
@@ -19,13 +19,6 @@
         HAITI, LIBERIA, SIERRA_LEONE, MEXICO, PERU, OTHER
     }
 
-    public enum Site {
```

Comment:
An alternative to this would have been to make Site an inner class, and then make the enum values into public String constants.  Then (I think) you could have left all of the code in place like ConfigDescriptor.Site.MIREBALAIS, as these would be String constants rather than enum values.  But it looks like this has been merged, so no worries either way @mogoodrich

Reply:
Yeah, I was thinking of something that left the variables in place as string constants, but it seemed like since (I think) we want to get away from the this kind of pattern in general (having direct references to specific sites within the code) I went with the  approach I did.

Reply:
Yep, definitely makes it more clear.